packages/react-ui-ag/src/Modals/__tests__/MoreOptionsModal-test.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import ThemedMoreOptionsModal from '../MoreOptionsModal'
+import theme from './mocks/theme'
+
+const MoreOptionsModal = ThemedMoreOptionsModal.WrappedComponent
+
+describe('MoreOptionsModal', () => {
+  const props = {
+    theme,
+    closeModal: jest.fn(),
+    filterTotal: 0,
+    total: 0,
+    isOpen: true,
+    onSubmit: jest.fn(),
+    clearFilters: jest.fn(),
+    restoreFilters: jest.fn(),
+  }
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('has header', () => {
+    const wrapper = mount(<MoreOptionsModal {...props} />)
+    const header = wrapper.find('.MoreOptionsModal_PropertyCount').at(0)
+    const headerText = header.text()
+
+    expect(header.length).toBe(1)
+    expect(headerText).toBe('0 Properties Found')
+  })
+
+  it('has close button', () => {
+    const wrapper = mount(<MoreOptionsModal {...props} />)
+    const closeButton = wrapper.find('.MoreOptionsModal_CloseButton').at(0)
+    const closeButtonText = closeButton.text()
+
+    closeButton.simulate('click')
+
+    expect(closeButton.length).toBe(1)
+    expect(closeButtonText).toBe('Close')
+    expect(props.restoreFilters).toHaveBeenCalledTimes(1)
+    expect(props.closeModal).toHaveBeenCalledTimes(1)
+    expect(props.onSubmit).not.toHaveBeenCalledTimes(1)
+    expect(props.clearFilters).not.toHaveBeenCalledTimes(1)
```

Comment:
So, this says that when you click on the close button, all of these get fired (4 things) ???